### ISSUE TITLE
errors when opening dashboard commands

### ISSUE THREAD

**Issue by samrath2007** [NTBBloodbath/doom-nvim#38]
```Error executing vim.schedule lua callback: ...scope.nvim/lua/telescope/previewers/buffer_previewer.lua:228: Vim(lua):E5108
: Error executing lua [string ":lua"]:1: attempt to index global 'Luasnip_current_nodes' (a nil value)```

**Comment by NTBBloodbath** [comment]
Hey, it looks like an error caused by `LuaSnip`. Is it installed? You can check it by doing `:PackerStatus` and looking for a `LuaSnip` entry :thinking:

**Comment by bdillahu** [comment]
Same error here... my :PackerStatus shows it loaded...

**Comment by NTBBloodbath** [comment]
Alright so I've found that it's a bug or a breaking change because it works when downgrading `LuaSnip`. I'll dig on it a bit more before pushing changes here. I'll let you know when it's fixed here.

Thank you for reporting this!

**Comment by NTBBloodbath** [comment]
Hey so I've uploaded a temporal fix for this behavior because a commit in `LuaSnip` broke the plugin (already reported it, let's wait for a fast fix).

So for temporarily fix this I have established that LuaSnip should use a specific commit (before the one that actually broke the code). The steps for fixing it will be:

1. Update Doom Nvim using `:DoomUpdate` command.
2. Remove the LuaSnip plugin (`rm -rf ~/.local/share/nvim/site/pack/packer/opt/LuaSnip`).
3. Launch Neovim to install `LuaSnip` again.
